internal/fs/vss_windows.go

```diff
@@ -811,14 +829,21 @@ func initializeVssCOMInterface() (*ole.IUnknown, error) {
 	}
 
 	// initialize COM security for VSS, this can't be called more then once
+	var capabilities uint32
+	if isPreWin8 {
+		capabilities = 0x00 // EOAC_NONE
+	} else {
+		capabilities = 0x20 // EOAC_STATIC_CLOAKING
+	}
+
 	if err = ole.CoInitializeSecurity(
 		-1,   // Allow *all* VSS writers to communicate back!
-		5,    // RPC_C_AUTHN_LEVEL_PKT_INTEGRITY
+		6,    // RPC_C_AUTHN_LEVEL_PKT_PRIVACY
 		3,    // RPC_C_IMP_LEVEL_IMPERSONATE
-		0x40, // EOAC_DYNAMIC_CLOAKING
+		capabilities,
 	); err != nil {
-		// TODO: warn for expected events for VSS failure event
-		fmt.Printf("VSS warning")
+		// TODO warn for expected event logs for VSS IVssWriterCallback failure
+		fmt.Printf("WARNING: failed to initialize security for VSS: %v\n", err)
```

Comment:
Setting the value to the recommended from the documentation(guessing they meant using `EOAC_STATIC_CLOAKING`), backing up with VSS works properly in Windows 11 and Windows 7(fixing the running issue with [hooking the api](https://github.com/Snshadow/go1.21-over-win7)). Should the value used for CoInitializeSecurity declared in the repository for readability?